tests/Alg/SignatureAlgorithmFactoryTest.php

```diff
@@ -39,23 +40,8 @@ public function setUp(): void
     public function testGetDigestAlgorithm(): void
     {
         $factory = new SignatureAlgorithmFactory([]);
-        $hmac = [
-            Constants::SIG_HMAC_SHA1      => Constants::DIGEST_SHA1,
-            Constants::SIG_HMAC_SHA224    => Constants::DIGEST_SHA224,
-            Constants::SIG_HMAC_SHA256    => Constants::DIGEST_SHA256,
-            Constants::SIG_HMAC_SHA384    => Constants::DIGEST_SHA384,
-            Constants::SIG_HMAC_SHA512    => Constants::DIGEST_SHA512,
-            Constants::SIG_HMAC_RIPEMD160 => Constants::DIGEST_RIPEMD160,
-        ];
-
-        $rsa = [
-            Constants::SIG_RSA_SHA1      => Constants::DIGEST_SHA1,
-            Constants::SIG_RSA_SHA224    => Constants::DIGEST_SHA224,
-            Constants::SIG_RSA_SHA256    => Constants::DIGEST_SHA256,
-            Constants::SIG_RSA_SHA384    => Constants::DIGEST_SHA384,
-            Constants::SIG_RSA_SHA512    => Constants::DIGEST_SHA512,
-            Constants::SIG_RSA_RIPEMD160 => Constants::DIGEST_RIPEMD160,
-        ];
+        $hmac = C::$HMAC_DIGESTS;
+        $rsa = C::$RSA_DIGESTS;
```

Comment:
Maybe we can even get rid of the variables and use `C::$WHATEVER` directly?

Reply:
Yes, please tell me how to name `WHATEVER` and I'd be glad to!  Considering this were two large lists of constants, I consider this a proper improvement